.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Debug settings so we can debug the tests in VS code. Launches debugger on currently open tests file.